### ISSUE TITLE
map not found issue

### ISSUE THREAD

**Issue by Teals53** [imayushsaini/Bombsquad-Ballistica-Modded-Server#104]
![image](https://github.com/user-attachments/assets/25af2005-2a7a-42b3-9845-d5db170d7c90)
what can i do for this ?

the maps inculded into scripts not seperated and works on standole but not on this server fork

**Comment by imayushsaini** [comment]
can you check @vishal332008 .

@Teals53 by standalone you mean a vanilla server ? which files you coping to vanilla server ? those file available here in this repository ?
Are you modifying those files before adding in vanilla server ? and what version of vanilla server and this modded server scipts you using ?

**Comment by Teals53** [comment]
> can you check @vishal332008 .
> 
> @Teals53 by standalone you mean a vanilla server ? which files you coping to vanilla server ? those file available here in this repository ? Are you modifying those files before adding in vanilla server ? and what version of vanilla server and this modded server scipts you using ?

yes for standole i mean vanilla server, the files i am copying to the server are the games who using custom maps inculded themself, names are demolotionwar memorygame ufo attack down into the abyss, yes those files available in this repo, no im not modifyng anything, i am using 1.7.36 version of vanilla server and latest current repo for ur server

**Comment by vishal332008** [comment]
> > can you check @vishal332008 .
> > @Teals53 by standalone you mean a vanilla server ? which files you coping to vanilla server ? those file available here in this repository ? Are you modifying those files before adding in vanilla server ? and what version of vanilla server and this modded server scipts you using ?
> 
> yes for standole i mean vanilla server, the files i am copying to the server are the games who using custom maps inculded themself, names are demolotionwar memorygame ufo attack down into the abyss, yes those files available in this repo, no im not modifyng anything, i am using 1.7.36 version of vanilla server and latest current repo for ur server

Are the maps shown in the error present in ba_root/mods/maps? If not present, add them to that folder.

**Comment by Teals53** [comment]
> > > can you check @vishal332008 .
> > > @Teals53 by standalone you mean a vanilla server ? which files you coping to vanilla server ? those file available here in this repository ? Are you modifying those files before adding in vanilla server ? and what version of vanilla server and this modded server scipts you using ?
> > 
> > 
> > yes for standole i mean vanilla server, the files i am copying to the server are the games who using custom maps inculded themself, names are demolotionwar memorygame ufo attack down into the abyss, yes those files available in this repo, no im not modifyng anything, i am using 1.7.36 version of vanilla server and latest current repo for ur server
> 
> Are the maps shown in the error present in ba_root/mods/maps? If not present, add them to that folder.

there is no maps the maps already inculded the scripts thats the reason already
